pkg/services/store/service.go

```diff
@@ -29,6 +29,15 @@ const RootResources = "resources"
 
 const MAX_UPLOAD_SIZE = 3 * 1024 * 1024 // 3MB
 
+type DeleteFolderCmd struct {
+	Path  string `json:"path"`
+	Force bool   `json:"force"`
+}
+
+type CreateFolderCmd struct {
+	Path string `json:"path"`
```

Comment:
Should we also add title?  
```suggestion
	Path string `json:"path"`
	Title string `json:"title"`
```

Reply:
I am not sure whether this is the right place for this:

- In general, I think it's better if the folder's name is consistent with its title, and the file name regex is not super restrictive so it should be possible in most of the cases - defaulting the title to file name should encourage this practice
- I assume we would keep using the folder's name rather than its title in the dataframe's `name` field & table & URL, so, in order to display it at all, we would need to add a separate folder-specific field to the dataframe 

wdyt about allowing the users to customize the title in the `configure` view or in some other operational UI?

Reply:
that works -- encouraging them to be the same makes sense to me.  The configure tab would be a good place to support setting a different title, or (😱 renaming to folder path)